controller/controller_server/exception_handler.py

```diff
@@ -0,0 +1,43 @@
+from decorator import decorator
+
+import grpc
+
+import controller.array_action.errors as controller_errors
+from controller.common.csi_logger import get_stdout_logger
+from controller.controller_server.errors import ObjectIdError
+from controller.controller_server.errors import ValidationException
+from controller.csi_general import csi_pb2
+
+logger = get_stdout_logger()
+
+exceptions_to_status_codes_map = {
```

Comment:
consider avoiding `map` by renaming to something like `status_codes_by_exception`

Reply:
Done

Reply:
I don't see any change

Reply:
I uploaded a commit recently.